cmd/cluster-ingress-operator/main.go

```diff
@@ -84,7 +84,7 @@ func main() {
 	}
 
 	// Set up the DNS manager.
-	dnsManager, err := createDNSManager(kubeClient, operatorNamespace, infraConfig, dnsConfig, clusterVersionConfig)
+	dnsManager, err := createDNSManager(kubeClient, operatorNamespace, infraConfig, dnsConfig)
```

Comment:
We can also delete the get at line 80, and the RBAC rule in `00-cluster-role.yaml`.

Reply:
Fixed